.github/workflows/test.yaml

```diff
@@ -48,6 +48,9 @@ jobs:
       - name: Lint and test
         run: npm test -- --color --maxWorkers=2
 
+      - name: Lint HTML
+        run: npm run lint:html
```

Comment:
Not a blocker, but confused me that `npm run lint` doesn't include `lint:html`

Should we copy the GOV.UK Frontend approach?

1. Run all lint tasks with `npm run lint`
2. Tests trigger build via `pretest`
3. Checks that need a build use `postbuild`

Reply:
It's not as scalable, but for now I've just added a `prelint:html` script that builds. That should run on `test`, `lint` and `lint:html` locally and during CI.

We can revisit if we ever decide we need to do anything more fancy.

Reply:
Forgive my cack-handedness at Github Actions!

I've removed the separate build step, but made it clear we're still building as part of the test step.